guilford_dps_offenders.yaml

```diff
@@ -0,0 +1,330 @@
+        # column: "OFFENDER ETHNIC CODE"
+        transforms:
+        - !<transforms.BooleanRegexTransform>
+          column: "OFFENDER ETHNIC CODE"
+          pattern: "(\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?)"
+          transformsIfTrue:
+          - !<transforms.ValueTransform>
+            value: null
+          transformsIfFalse:
+          - !<transforms.ColumnTransform>
+            column: "OFFENDER ETHNIC CODE"
+          - !<transforms.ReplaceTransform>
+            target: ["EUROPEAN/N.AM./AUSTR", "AMERICAN INDIAN", "SLAVIC (E. EUROPEAN)", "ORIENTAL", "OTHER", "ASIAN", "UNKNOWN", "HISPANIC/LATINO", "AFRICAN", "NORDIC/SCANDANAVIAN", "PACIFIC ISLANDER"]
+            goal: ["white", "amindian", "white", "asian", "other", "asian", "", "", "black", "white", "pacisland" ]
+      nc.PersonEthnicity:
+        type: "nc.PersonEthnicity"
+        # column: "OFFENDER ETHNIC CODE"
+        transforms:
+        - !<transforms.BooleanRegexTransform>
+          column: "OFFENDER ETHNIC CODE"
+          pattern: "(\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?)"
+          transformsIfTrue:
+          - !<transforms.ValueTransform>
+            value: null
+          transformsIfFalse:
+          - !<transforms.ColumnTransform>
+            column: "OFFENDER ETHNIC CODE"
+          - !<transforms.ReplaceTransform>
+            target: ["EUROPEAN/N.AM./AUSTR", "AMERICAN INDIAN", "SLAVIC (E. EUROPEAN)", "ORIENTAL", "OTHER", "ASIAN", "UNKNOWN", "HISPANIC/LATINO", "AFRICAN", "NORDIC/SCANDANAVIAN", "PACIFIC ISLANDER"]
+            goal: ["nonhispanic", "nonhispanic", "nonhispanic", "nonhispanic", "nonhispanic", "nonhispanic", "", "hispanic", "nonhispanic", "nonhispanic", "nonhispanic"]
+      ol.datasource:
+        type: "ol.datasource"
+        transforms:
+        - !<transforms.ValueTransform>
+          value: "Guilford Co. DPS"
+    name: "offender"
+
+  offenderappearance:
+    fqn: "ol.physicalappearance"
+    entitySetName: "GuilfordOffenderPhysicalAppearances"
+    propertyDefinitions:
+      general.id:
+        type: "general.id"
+        column: "OFFENDER NC DOC ID NUMBER"
+      ol.height:
+        type: "ol.height"
+        column: "OFFENDER HEIGHT (IN INCHES)"
+      ol.weight:
+        type: "ol.weight"
+        column: "OFFENDER WEIGHT (IN LBS)"
+      nc.complexion:
+        type: "nc.complexion"
+        # column: "OFFENDER SKIN COMPLEXION CODE"
+        transforms:
+        - !<transforms.BooleanRegexTransform>
+          column: "OFFENDER SKIN COMPLEXION CODE"
+          pattern: "(\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?\\?)"
+          transformsIfTrue:
+          - !<transforms.ValueTransform>
+            value: null
+          transformsIfFalse:
+          - !<transforms.ColumnTransform>
+            column: "OFFENDER SKIN COMPLEXION CODE"
+          - !<transforms.ReplaceTransform>
```

Comment:
this replace transform is fine, but unneeded. You could just read it in as is - or use the case transform instead.